src/board_controller/brainalive/brainalive.cpp

```diff
@@ -190,6 +192,7 @@ int BrainAlive::prepare_session ()
     if ((res == (int)BrainFlowExitCodes::STATUS_OK) && (control_characteristics_found))
     {
         initialized = true;
+        config_board ("0a036007000d");
```

Comment:
Add check for result here

Reply:
So it should be res =... and below that initialized = true

Reply:
done